sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/AmqpClientOptions.java

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.amqp;
+
+import com.azure.core.util.ClientOptions;
+import com.azure.core.util.Header;
+
+/**
+ * A set of AMQP client options.
```

Comment:
May improve the javadoc on the class.

E.g. https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/core/azure-core/src/main/java/com/azure/core/util/HttpClientOptions.java#L20

Suggestion for discussion, "General configuration options for AMQP Clients."?

Reply:
It's better. I referred the `AmqpRetryOptions` before.